src/Makefile

```diff
@@ -1,9 +1,27 @@
 MAKEFLAGS=--no-print-directory --section-alignment 0x1000 -I$(PWD)
 #$(info Make flags $(MAKEFLAGS))
 
+.PHONY: default all extern comps plat init_extern
 default: | all cp
 
-all: comps plat
+all: | extern comps plat
+
+extern:
+	$(info )
+	$(info ***********************************************)
+	$(info *********** Building External Libs ************)
+	$(info ***********************************************)
+	$(info )
+	$(shell cd extern && python make.py >&2)
+
+init_extern:
+	$(info )
+	$(info ***********************************************)
+	$(info ********** Setting up External Libs ***********)
+	$(info ***********************************************)
+	$(info )
+	bash -c 'cd extern/cFE/; source setvars.sh; cd ../..; make -C extern/cFE/build/cpu1 config'
```

Comment:
We can hide all the python specific stuff inside a `cFS` subdir.  See comments later.